src/app/exhibitor/_components/TimelineItem.tsx

```diff
@@ -0,0 +1,46 @@
+import { P } from "@/app/_components/Paragraph"
+import { cn } from "@/lib/utils"
+import {
+	AccordionItem,
+	AccordionContent,
+	AccordionTrigger
+} from "@/components/ui/accordion"
+
+export function TimelineItem({
+	children,
+	title,
+	dateString
+}: {
+	children?: React.ReactNode
+	title: string
+	dateString: string
+}) {
+	const expandable = children != null
+	return (
+		<AccordionItem
+			key={title}
+			value={title}
+			disabled={!expandable}
+			className="border-none">
+			<div className="absolute -start-1.5 mt-3.5 h-3 w-3 rounded-full border border-white bg-melon-700"></div>
+			<AccordionTrigger
+				disabled={!expandable}
+				className={cn(
+					"mb-2 ml-4 w-full rounded px-2 pb-4 pt-0 text-left font-normal hover:no-underline",
+					expandable && "transition hover:bg-slate-700"
```

Comment:
this works, but the preferred convention here would be:

```tsx
cn("mb-2 ml-4 w-full rounded px-2 pb-4 pt-0 text-left font-normal hover:no-underline", {
    "transition hover:bg-slate-700": expandable
}}
```

this is the same thing logically but a little more readable